ROADMAP.md

```diff
@@ -25,3 +25,62 @@ Feeature ideas for the OpenSCD project. They are currently not prioritized.
+     - to have an API for GOOSE/SV publishing/subscriptions and the ability to instantiate/modify LGOS and LSVS LNs.
+   - Generate logic used for interlocking (Transpower uses a somewhat novel "topology-based" interlocking based off an SSD file which generate IEC61131 logic).
+
+1. SLD Editor
+
+   - Transpower wishes to carry out "top-down engineering" based on the SLD editor. The Substation editor in OpenSCD already provides much of the capability required.
+   - However the following are outstanding:
+     - Drag and drop of SLD symbols to create elements (from voltage level, bay, transformer, conducting equipment etc.) (including the `sxy` namespace coordinates)
+     - Ability to edit key parameters (voltage levels, descriptions)
+     - Ability to rotate symbols
+     - Ability to group objects, copy and paste and carry out basic alignment tasks (or to be connectable on a grid)
+     - Ability to "flip" a group of objects (to allow a bay to drawn in a particular direction -- Transpower draws bays to match the physical orientation of the site)
+     - Ability to verify/trace connectivity
+
+1. Substation Editor. The Substation editor provides much of the desired capability but does not yet allow:
+
+   - Connection of a an IED to a set of LNodes. (see [#1128](https://github.com/openscd/open-scd/issues/1128))
+   - Ability to model the NeutralPoint of a transformer. Transpower needs to be able to assign CTRs to the neutral point of the transformer so we need to be able to model this ([#796](https://github.com/openscd/open-scd/issues/796))
+
+1. General SCL diffing/merging tool -- this is quite a significant piece of work but is important to allow:
+
+   - Updating of an SCD file from IID file provided by an ICT. This is needed for the round trip from e.g. DIGSI 5 where the initial ICD file is inadequate and a bay specific IID file must be used. We would like to update the IED in the system configuration tool (generated from a ICD file) with the IID from the ICT. There is always going to be a need for customisation/update of specific devices which don't match the template ICD files. (This is S110 of Table G.2 in IEC 61850-6 Ed 2.1).
+
+   - Being able to verify changes quickly and efficiently and handle collisions/merge differences if multiple people are working on the same project (currently we would plan to use git and plain-text differencing for management of SCL files)
+
+   - Being able to exchange one IED with another with a slightly different data model
+
+   Some initial prototyping and ideating done by Daniel/Christian (see also: [#896](https://github.com/openscd/open-scd/issues/896), [#669](https://github.com/openscd/open-scd/issues/669), [#892](https://github.com/openscd/open-scd/issues/892), [#349](https://github.com/openscd/open-scd/issues/349))
+
+1. Complete the publisher plugin 
+
+   - Provide the ability to add FCDOs to RCBs to allow MMS SCADA map augmentation ([#1092](https://github.com/openscd/open-scd/issues/1092))
```

Comment:
```suggestion
   - Provide the ability to add FCDAs to RCBs to allow MMS SCADA map augmentation ([#1092](https://github.com/openscd/open-scd/issues/1092))
```

Reply:
I think it's actually FCDOs. We can already do FCDAs in OpenSCD, but we can't drop entire data objects in like we do in other tools.

Reply:
Ah alright. FCDO isn't mentioned within IEC 61850, so some may get confused. But it looks like other third parties have started using that term to define a functionally constrained data object.

This is the definition of FCDA:

> A reference to a single data attribute, a sub data object or sub data attribute of a data object having a specific functional constraint (FC) value shall be called functional constrained data attribute (FCDA).